dlt/extract/hints.py

```diff
@@ -202,8 +263,13 @@ def apply_hints(
                 )
                 if schema_contract is not None:
                     t["schema_contract"] = schema_contract
+            if table_format is not None:
```

Comment:
this is kind of unrelated, but what about a utility function that does all of the table_format updating (and all other props that behave the same way) in one line? this makes it all more readable and you can also use it in make_hints (although the popping part is not needed there really)